docs/website/docs/dlt-ecosystem/verified-sources/index.md

```diff
@@ -1,31 +1,45 @@
 ---
-title: Verified sources
-description: List of verified sources
-keywords: ['verified source']
+title: Sources
```

Comment:
one note about this page: we also have a couple of templates now as demonstrated on thursday. @AstrakhantsevaAA knows all about this :)